### PR TITLE
Increased code coverage for src/components/AddOn/core/AddOnStore/AddOnStore.tsx file

### LINKED ISSUE #1038: Code Coverage: Create tests for src/components/AddOn/core/AddOnStore/AddOnStore.tsx
We will need unittests done for all methods, classes and functions found in this file.

Any widgets, components, modals referenced in this file must also have unittests done

### PR Acceptance Criteria

- When complete this file must show at least **100%** coverage when merged into the code base. 
- [The current code coverage for the file can be found here](https://app.codecov.io/gh/PalisadoesFoundation/talawa-admin?search=&displayType=list)
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide
- Please coordinate **issue assignment** and **PR reviews** with the contributors listed in this issue https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Anvita0305**
**What kind of change does this PR introduce?**
Added tests for increasing the code coverage of the file to 100%

**Issue Number:**

Fixes #1038 

**Did you add tests for your changes?**
N/A

**Snapshots/Videos:**
Current code coverage:
Screenshot from 2024-02-22 09-22-29

**If relevant, did you update the documentation?**
N/A

**Summary**
It increases the code coverage to 100% which was initially 38%.

**Does this PR introduce a breaking change?**
No.

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by Cioppolo14** [comment]
@Anvita0305 Can you fix any introspection errors related to your changes?

**Comment by Cioppolo14** [comment]
@noman2002 @beingnoble03 Can you review this PR?

**Comment by Anvita0305** [comment]
> @Anvita0305 Can you fix any introspection errors related to your changes?

Hey @Cioppolo14 , sorry for late reply. I think this introspection test which is not this PR specific. However, if the issue for it is not yet created, I can create it.

File: src/components/AddOn/core/AddOnStore/AddOnStore.tsx [hunk 1]
```diff
@@ -266,7 +260,9 @@ function addOnStore(): JSX.Element {
                 </Tab>
                 <Tab eventKey="installed" title={t('install')}>
                   {data.getPlugins
-                    .filter((plugin: any) => plugin.pluginInstallStatus == true)
+                    .filter(
+                      (plugin: any) => !plugin.uninstalledOrgs.includes(orgId),
+                    )
```
Comment (beingnoble03) [inline src/components/AddOn/core/AddOnStore/AddOnStore.tsx:265]: @Anvita0305 Can you please explain why this is changed? (cc @SiddheshKukade is this a required change?)

**Review by beingnoble03 — COMMENTED** [review]
@SiddheshKukade please look into this.

File: src/components/AddOn/core/AddOnStore/AddOnStore.tsx (hunk 1 above)
Comment (Anvita0305) [reply src/components/AddOn/core/AddOnStore/AddOnStore.tsx:265]: Initially, plugin.pluginInstallStatus was being filtered if true. But in the latest schema there is no such field. 
So instead, I am checking if the current orgId is not in uninstalledOrgs list of the plugins.

File: src/components/AddOn/core/AddOnStore/AddOnStore.tsx [hunk 2]
```diff
@@ -76,13 +72,13 @@ function addOnStore(): JSX.Element {
...
   };
   // /* istanbul ignore next */
-  const pluginModified = (): void => {
-    return getInstalledPlugins();
-    // .then((installedPlugins) => {
-    //   getStorePlugins();
-    //   return installedPlugins;
-    // });
-  };
+  // const pluginModified = (): void => {
```
Comment (beingnoble03) [inline src/components/AddOn/core/AddOnStore/AddOnStore.tsx:75]: I think this is not a valid function, as you said `pluginInstallStatus ` is no longer supported. I think we can completely remove this func?

Comment (Anvita0305) [reply src/components/AddOn/core/AddOnStore/AddOnStore.tsx:75]: Yeah its not a valid function. I will remove the commented code and update the file.

Comment (Anvita0305) [reply src/components/AddOn/core/AddOnStore/AddOnStore.tsx]: Hello, I have updated the file.

**Review by palisadoes — APPROVED** [review]

**Pull request merged**